Translations/translation_NL.json

```diff
@@ -1,207 +1,339 @@
-    },
-    "ScrollingSpeed": {
-      "text2": ["Scroll", "snelheid"],
-      "desc": "Snelheid waarmee de tekst scrolt."
-    },
-    "QCMaxVoltage": {
-      "text2": ["Power", "Wattage"],
-      "desc": "Wattage van de gebruikte voeding"
-    },
-    "PDNegTimeout": {
-      "text2": ["PD", "timeout"],
-      "desc": "PD negotiation timeout in 100ms steps for compatibility with some QC chargers (0: disabled)"
-    },
-    "PowerLimit": {
-      "text2": ["Vermogen", "limiet"],
-      "desc": "Maximaal vermogen (Watts)"
-    },
-    "ReverseButtonTempChange": {
-      "text2": ["Draai", "+ - knoppen om"],
-      "desc": "Keer de +- knoppen om de termperatuur van de punt te regelen om."
-    },
-    "TempChangeShortStep": {
-      "text2": ["Temp veranderen", "kort"],
-      "desc": "Temperatuur verandering bij kort drukken"
-    },
-    "TempChangeLongStep": {
-      "text2": ["Temp veranderen", "lang"],
-      "desc": "Temperatuur verandering bij lang drukken"
-    },
-    "PowerPulsePower": {
-      "text2": ["Vermogen", "Puls W"],
-      "desc": "Vermogen van puls om soldeerbout aan te houden"
-    },
-    "HallEffSensitivity": {
-      "text2": ["Hall sensor", "sensitivity"],
-      "desc": "Sensitivity of the Hall effect sensor to detect sleep (0=uit | 1=minst gevoelig | ... | 9=meest gevoelig)"
-    },
-    "LockingMode": {
-      "text2": ["Blokkeer", "knoppen"],
-      "desc": "Tijdens solderen lang op beide knoppen drukken, blokkeert ze. (U=Uit | B=Alleen boost | V=Volledig blokkeren)"
-    },
-    "MinVolCell": {
-      "text2": ["Minimum", "voltage"],
-      "desc": "Minimum allowed voltage per cell (3S: 3 - 3.7V | 4-6S: 2.4 - 3.7V)"
-    },
-    "AnimLoop": {
-      "text2": ["Anim.", "loop"],
-      "desc": "Loop icon animations in main menu"
-    },
-    "AnimSpeed": {
-      "text2": ["Anim.", "speed"],
-      "desc": "Pace of icon animations in menu (O=off | S=slow | M=medium | F=fast)"
-    },
-    "PowerPulseWait": {
-      "text2": ["Power pulse", "delay"],
-      "desc": "Delay before keep-awake-pulse is triggered (x 2.5s)"
-    },
-    "PowerPulseDuration": {
-      "text2": ["Power pulse", "duration"],
-      "desc": "Keep-awake-pulse duration (x 250ms)"
-    },
-    "LanguageSwitch": {
-      "text2": ["Spraak :", " NL  Nederlands"],
-      "desc": ""
-    },
-    "Brightness": {
-      "text2": ["Screen", "brightness"],
-      "desc": "Adjust the brightness of the OLED screen"
-    },
-    "ColourInversion": {
-      "text2": ["Invert", "screen"],
-      "desc": "Invert the colours of the OLED screen"
-    },
-    "LOGOTime": {
-      "text2": ["Boot logo", "duration"],
-      "desc": "Sets the duration for the boot logo (s=seconds)"
-    }
-  }
-}
+{
+ "languageCode": "NL",
+ "languageLocalName": "Nederlands",
+ "fonts": [
+  "ascii_basic",
+  "latin_extended"
+ ],
+ "tempUnitFahrenheit": false,
+ "messages": {
+  "SettingsCalibrationWarning": "Zorg ervoor dat te punt op kamertemperatuur is voor je verder gaat!",
```

Comment:
"Zorg ervoor dat de punt op kamertemperatuur is voor je verder gaat!"

Reply:
Good catch! Fixed it.